src/Umbraco.Cms.Api.Management/Controllers/AuditLog/ByKeyAuditLogController.cs

```diff
@@ -28,7 +28,7 @@ public ByKeyAuditLogController(IAuditService auditService, IAuditLogPresentation
     [HttpGet("{id:guid}")]
     [MapToApiVersion("1.0")]
     [ProducesResponseType(typeof(PagedViewModel<AuditLogResponseModel>), StatusCodes.Status200OK)]
-    public async Task<IActionResult> ByKey(Guid id, Direction orderDirection = Direction.Descending, DateTime? sinceDate = null, int skip = 0, int take = 100)
+    public async Task<IActionResult> ByKey(CancellationToken cancellationToken, Guid id, Direction orderDirection = Direction.Descending, DateTime? sinceDate = null, int skip = 0, int take = 100)
```

Comment:
Just commenting on this once (it seems to apply to all changes), but I would move all these to the end so it doesn't violate CA1068:
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1068

Reply:
Thanks for your concern, given the special case: 

```
Method has one or more [optional](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/named-and-optional-arguments#optional-arguments) parameters ([Optional](https://learn.microsoft.com/en-us/dotnet/visual-basic/programming-guide/language-features/procedures/optional-parameters) in Visual Basic) following a non-optional cancellation token parameter. The compiler requires that all optional parameters are defined after all the non-optional parameters.
```

I'd opt for consistency, as in it consistently being the first parameter, rather than sometimes being the last, and sometimes being somewhere in the middle.

However, I'll bring this up with the team to see what they think as well.

Reply:
I think it makes sense to make CancellationToken optional, as that's quite common.

Reply:
I just talked with the team, and we all agreed that since these are controllers, and thus won't be invoked like a normal method then ca1068 is not relevant to us in this case, with that in mind and the thing I mentioned with optional parameters we've decided that we prefer having it as the first parameter, thanks for your concern though 🙌